test/rules/typedef-whitespace/both/onespace/test.ts.lint

```diff
@@ -220,7 +220,7 @@ function withPreceedingSpaceNoSubsequentSpaceFn(a :number, b :number) :number {
     }
 }
 
-function withPreceedingSpaceWithSubsequentSpaceFn(a : number, b : number) : number {
+function withPreceedingSpaceWithSubsequentSpaceFn(a : number, b ?: number) : number {
```

Comment:
I took a survey of random TS developers, and found that this syntax wasn't really used elsewhere. (Overwhelmingly people used `varName?: type`.) So unfortunately that makes this PR a change that's not appropriate for core TSLint. However, I would definitely recommend making a custom rule and using it at your company! Obviously you're welcome to use the current rule in TSLint as your starting point